src/modules/automatic_tags.js

```diff
@@ -5,11 +5,11 @@ module.exports = {
                 type: "Automatic"
             })
             .then((data) => {
-                    for (let i = 0; i < data.length; i++) {
-                        let res = new RegExp(data[i].title, 'gi');
-                        let tag = subject.match(res);
-                        callback(tag);
+                for (let i = 0; i < data.length; i++) {
+                    let res = new RegExp(data[i].title, 'gi');
+                    let tag = subject.match(res);
+                    callback(tag);
```

Comment:
this doesn't make sense to do callback inside forloop. this will call callback() many times 

also use promise only, no callbacks